codegen/core/src/main/java/software/amazon/smithy/python/codegen/writer/MarkdownToRstDocConverter.java

```diff
@@ -71,47 +71,73 @@ public void head(Node node, int depth) {
                 TextNode textNode = (TextNode) node;
                 String text = textNode.text();
                 if (!text.trim().isEmpty()) {
-                    sb.append(text);
-                    // Account for services making a paragraph tag that's empty except
-                    // for a newline
+                    if (text.startsWith(":param ")) {
+                        int secondColonIndex = text.indexOf(':',  1);
+                        writer.write(text.substring(0, secondColonIndex + 1));
+                        //TODO right now the code generator gives us a mixture of
+                        // commonmark and HTML (for instance :param xyz: <p> docs
```

Comment:
Minor clarification - [inline HTML is valid CommonMark](https://spec.commonmark.org/0.29/#html-blocks).